rtengine/imageio.cc

```diff
@@ -83,6 +85,43 @@ Glib::ustring to_utf8 (const std::string& str)
     }
 }
 
+unsigned short addSat (unsigned short a, unsigned short b)
```

Comment:
The tricky part in a vectorized version will be the access to the 'plusses' lookup table which also is the reason why (according to current processing time) it does not get auto vectorized. But I have to look at the assembly to verify